tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -33,7 +33,16 @@
 
 
 @pytest.mark.parametrize(
-    "deployment_type, per_unit_storage_throughput", [("PERSISTENT_1", 200), ("SCRATCH_1", None), ("SCRATCH_2", None)]
+    "deployment_type, per_unit_storage_throughput, storage_type, drive_cache_type, storage_capacity",
+    [
+        ("PERSISTENT_1", 200, None, None, 1200),
+        ("SCRATCH_1", None, None, None, 1200),
+        ("SCRATCH_2", None, None, None, 1200),
+        ("PERSISTENT_1", 200, "SSD", None, 1200),
+        ("PERSISTENT_1", 40, "HDD", "NONE", 1800),
+        ("PERSISTENT_1", 40, "HDD", "READ", 1800),
```

Comment:
I think we can avoid this test, it's very similar to the following one.

Reply:
I am not sure. 
For per_unit_storage_throughput is 40, the storage_capacity should be the multiple of 1800
For per_unit_storage_throughput is 12, the storage_capacity should be the multiple of 6000
Should we delete one test case?

Reply:
In our tests we should verify that the features are working correctly within ParallelCluster but we don't need to test all the combination of the FSx parameters, it is up to the FSx team to verify that the all the combinations work.

In the unit tests it is good to have all the combinations because we're testing our validators but for integration tests we can remove the `("PERSISTENT_1", 40, "HDD", "READ", 1800)` and just keep the `("PERSISTENT_1", 12, "HDD", "READ", 6000),` because we're already testing the 40 case in the pervious test.

Reply:
Oh, I see! Done!